### PR TITLE
Milestone3 events

### PR DESCRIPTION
**Pull request by blakbox23**
In this part of the project I : 
Added `operate.js` and `calculate.js` components to the project
Gave the buttons event handlers to facilitate the working of the calculator.
Used event binding to allow for change of state after the click event.

### CODE REVIEW

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+ *   operation:String  +, -, etc.
+ */
+export default function calculate(obj, buttonName) {
+  if (buttonName === 'AC') {
+    return {
+      total: null,
+      next: null,
+      operation: null,
+    };
+  }
```
Comment (DeliceLydia) [inline src/logic/calculate.js]: - [x] I suggest that when you click on this button `AC` to display a default `zero in the screen

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @blakbox23, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/logic/calculate.js (hunk 1 above)
Comment (blakbox23) [reply src/logic/calculate.js]: thanks @DeliceLydia, I just needed to set the default of result as 0 in `calculate.js` 👍🏾

**Review by george-shammar — APPROVED** [review]
Hi @blakbox23 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**